troubleshooting/tools/tracevol.py

```diff
@@ -58,16 +61,24 @@
                     default="admin", help="user ID to connect to ceph cluster")
 PARSER.add_argument("-key", "--userkey",
                     default="", help="user password to connect to ceph cluster")
+PARSER.add_argument("-cm", "--configmap", default="ceph-csi-config",
+                    help="config map where subvolumegroup is specified")
```

Comment:
update the help, as this will be used for rbd also in future, better to say configmap name which holds the cephcsi configuration?